pkgs/development/python-modules/raylib-python-cffi/default.nix

```
@@ -0,0 +1,56 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cffi,
+  pkg-config,
+  glfw,
+  libffi,
+  raylib,
+  physac,
+  raygui,
+  lib
+}:
+
+buildPythonPackage rec {
+  pname = "raylib-python-cffi";
+  version = "5.0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "electronstudio";
+    repo = "raylib-python-cffi";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DlnZRJZ0ZnkLii09grA/lGsJHPUYrbaJ55BVWJ8JzfM=";
+  };
+
+  build-system = [
+    setuptools
+    cffi
+  ];
+
+  patches = [
+    # This patch fixes to the builder script functio to call pkg-config
+    # using the library name rather than searching only through raylib
+    ./fix_pyray_builder.patch
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  # tests require a graphic environment
+  doCheck = false;
+
+  buildInputs = [
+    glfw
+    libffi
+    raylib
+    physac
+    raygui
+  ];
+
+  meta = {
+    description = "Python CFFI bindings for Raylib.";
```

```suggestion
    description = "Python CFFI bindings for Raylib";
```
Please remove a trailing period.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description